integration

```diff
@@ -113,6 +113,7 @@
   "bacco007/sensor.waternsw",
   "barban-dev/homeassistant-midea-dehumidifier",
   "barleybobs/homeassistant-ecowater-softener",
+  "barreeeiroo/Home-Assistant-Electric-Ireland",
```

Comment:
For a specific country, missing country key in released hacs.json

Reply:
@ludeeus apologies for that, it's now fixed. I have added the IE country flag to `hacs.json`.

Reply:
https://github.com/barreeeiroo/Home-Assistant-Electric-Ireland/blob/v0.1.0/hacs.json this is the released version.

Reply:
Fixed, released a new version and updated the manifest:

* https://github.com/barreeeiroo/Home-Assistant-Electric-Ireland/blob/v0.1.1/hacs.json
* https://github.com/barreeeiroo/Home-Assistant-Electric-Ireland/blob/v0.1.1/custom_components/electric_ireland_insights/manifest.json